index.html

```diff
@@ -110,7 +98,122 @@ <h3 id="gold-sidebar">
+            </li>
+            <li>
+              <a href="#location">Location</a>
+            </li>
+            <li>
+              <a href="#program">Program</a>
+            </li>
+          </ul>
+        </nav>
+        <section id="intro">
+          <h2 id="agenda">
+            Agenda
+          </h2>
+          <style type="text/css">
+            .tg  {border-collapse:collapse;border-spacing:0;}
+            .tg td{font-family:Arial, sans-serif;font-size:14px;padding:10px 5px;border-style:solid;border-width:1px;overflow:hidden;word-break:normal;}
+            .tg th{font-family:Arial, sans-serif;font-size:14px;padding:10px 5px;border-style:solid;border-width:1px;overflow:hidden;word-break:normal;}
+            .tg .tg-s6z2{text-align:center}
+            .tg .tg-baqh{text-align:center;vertical-align:top}
+            .tg th.empty, .tg td.empty { border: none; }
+            .tg .lunch { background-color: rgb(240, 255, 215); }
+            .tg .break { background-color: rgb(255, 231, 179); }
+            .tg .org   { background-color: rgb(236, 236, 236); }
+            .tg .plenary { background-color: rgb(191, 242, 255); }
+            .tg .followup { background-color: rgb(174, 186, 255); }
+            .tg .exhibition { background-color: rgb(255, 208, 208); }
+            .tg .digest { background-color: rgb(255, 210, 255); }
+          </style>
+          <table class="tg">
+            <tr>
+              <th class="empty"></th>
+              <th class="tg-s6z2" colspan="2">Tuesday, 5</th>
+              <th class="tg-s6z2" colspan="2">Wednesday, 6</th>
+              <th class="tg-s6z2">Thursday, 7</th>
+            </tr>
+            <tr>
+              <th class="tg-s6z2" rowspan="3">Morning</th>
+              <td class="tg-baqh org" colspan="2">Registration</td>
+              <td class="tg-s6z2 plenary" colspan="2" rowspan="3">Plenary sessions</td>
+              <td class="tg-s6z2 digest" rowspan="2">Digest</td>
+            </tr>
+            <tr>
+              <td class="tg-baqh org" colspan="2">Welcome</td>
+            </tr>
+            <tr>
+              <td class="tg-s6z2 plenary" colspan="2">Plenary sessions</td>
+              <td class="tg-baqh org">Opportunities &amp; challenges</td>
+            </tr>
+            <tr>
+              <td class="tg-baqh break" colspan="6">Coffee break</td>
+            </tr>
+            <tr>
+              <th class="tg-baqh">Morning</th>
+              <td class="tg-baqh plenary" colspan="2">Plenary sessions</td>
+              <td class="tg-baqh plenary" colspan="2">Plenary sessions</td>
+              <td class="tg-baqh org">Closure</td>
+            </tr>
+            <tr>
+              <td class="tg-baqh lunch" colspan="5">Lunch</td>
+              <td class="empty" rowspan="4"></td>
+            </tr>
+            <tr>
+              <th class="tg-s6z2">Afternoon</th>
+              <td class="tg-s6z2 followup">Folllow-up sessions</td>
```

Comment:
I'm not sure how to handle this within the current proposal, but as I mentioned by email, one of my accessibility colleagues can only join us on Monday afternoon - it would be great if we could make sure this gets discussed then.

Reply:
Yes, will do. This is a preliminar agenda, just to broadly describe what we will be doing. Does it make sense?